### PR TITLE
Add find_sql_cases_not_updated_in_es command

### PR DESCRIPTION
**Pull request by dannyroberts**
##### SUMMARY
and make updates to `find_sql_forms_not_in_es`

### CODE REVIEW

File: corehq/apps/data_pipeline_audit/management/commands/find_sql_cases_not_updated_in_es.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                print(f'{info.case_id}\t{info.server_modified_on}\t{info.domain}')
+            if fix:
+                for case in CaseAccessorSQL.get_cases([info.case_id for info in bad_set]):
+                    publish_case_saved(case, send_post_save_signal=False)
+
+
+def get_case_ids_chunked(domain=None, start_date=None, end_date=None):
+    reindex_accessor = CaseReindexAccessor(
+        domain=domain, start_date=start_date, end_date=end_date)
+    yield from iter_all_ids_chunked(reindex_accessor)
```
Comment (dannyroberts) [inline corehq/apps/data_pipeline_audit/management/commands/find_sql_cases_not_updated_in_es.py]: These queries take a really long time for some reason. The initial chunk takes about 30 seconds and (maybe sometimes a lot longer?) to fetch 500 cases, which seems really slow to me. The query I tried was `get_case_ids_chunked(domain=None, start_date=date(2020, 2, 1), end_date=date(2020, 3, 31))`.  @snopoke have you found `CaseReindexAccessor` to be slow in general, or might I be doing something especially wrong here?

**Comment by czue** [comment]
I didn't look closely at this, but it sounds a lot like the `stale_data_in_es` command. did you look at / are you aware of it?

https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqadmin/management/commands/stale_data_in_es.py

**Comment by dannyroberts** [comment]
Wow, thank you @czue it's exactly that. I'm going to want to make that command (at least for the SQL backend) support not filtering by domain and support large result sets through pagination, but the general point of the command is exactly what I'm trying to do

**Comment by czue** [comment]
:+1: cool. we used it on ICDS so I think it should support large result sets.

**Comment by dannyroberts** [comment]
Replacing this with https://github.com/dimagi/commcare-hq/pull/26961

**Pull request closed without merging**